src/helper.js

```diff
@@ -127,7 +127,7 @@ var marexandre;
      * @return {Array}                 [description]
      */
     Helper.prototype.cleanupOnWordBoundary = function(text, list, useWordBoundary) {
-      useWordBoundary = useWordBoundary || true;
+      useWordBoundary = !!useWordBoundary;
```

Comment:
doesn't this remove the default `true`? 🤔

Reply:
since other parameters also have default value and I would keep code in minimum changes. WDYT?

Reply:
where is the default value? I don't see it. But LGTM if it works

Reply:
it's here. sorry, it's in different file.
https://github.com/gengo/jquery.textarea-highlighter.js/blob/55c654b9ec1f0caa4d09ee28d48ddb30dd3867a8/src/jquery.textarea-highlighter.js#L22